distsql/request_builder.go

```diff
@@ -104,6 +104,7 @@ func (builder *RequestBuilder) SetChecksumRequest(checksum *tipb.ChecksumRequest
 		builder.Request.StartTs = checksum.StartTs
 		builder.Request.Data, builder.err = checksum.Marshal()
 		builder.Request.NotFillCache = true
+		builder.Request.Priority = kv.PriorityLow
```

Comment:
Do not set a priority to bypass the priority in the statement context. It could be user-defined.

Reply:
That's a very good point! I'm closing it now.